### PR TITLE
Add Users support 

### PR DESCRIPTION
**Pull request by lukabratos**
### What?

Offer a secure method to directly target a notification to an authenticated user in your application.

### Why?

This can be useful if you want to send personalized notifications, for example, if you have an events application, you could send invites targeting specific users knowing that these notifications can’t be accessed by other users.

### CODE REVIEW

**Comment by lukabratos** [comment]
- [ ] Set SDK version to `1.0.0` and update the `README.md`

File: Sources/PushNotifications/PushNotifications.swift [hunk 1]
```diff
@@ -85,75 +99,354 @@ public struct PushNotifications {
...
+    let publishRequest = [
+        "apns": [
+            "aps": [
+                "alert": "Hello"
+            ]
+        ],
+        "fcm": [
+            "notification": [
+                "title": "Hello",
+				"body":  "Hello, world",
```
Comment (luismfonseca) [inline Sources/PushNotifications/PushNotifications.swift]: too much tabbing

Comment (lukabratos) [reply Sources/PushNotifications/PushNotifications.swift]: Fixed in 30490bae731aed993b9437b9774f3097b0c5146e

File: README.md [hunk 2]
```diff
@@ -81,10 +81,10 @@ pushNotifications.publishToUsers(["jonathan", "jordan", "luís", "luka", "mina"]
 })
 
 // Authenticate User
-pushNotifications.authenticateUser("Elon M", completion: { result in
+pushNotifications.generateToken("Elon M", completion: { result in
     switch result {
-    case .value(let jwtTokenString):
-        print("\(jwtTokenString)")
+    case .value(let jwtToken):
+        print("\(jwtToken)")
```
Comment (luismfonseca) [inline README.md]: How will this print now that its a dictionary?

Comment (luismfonseca) [reply README.md]: e.g. Do we have any language built-in way to suggest them to marshal to JSON?

Comment (lukabratos) [reply README.md]: This is just an example.

**Pull request merged**